### PR TITLE
chore!: bump required node version to 16

### PR DESCRIPTION
**Pull request by jcesarmobile**

### CODE REVIEW

File: packages/@ionic/utils-subprocess/src/index.ts [hunk 1]
```diff
@@ -187,16 +187,16 @@ export class Subprocess {
 
         if (code === 0) {
           return resolve();
-        }
-
-        if (signal) {
+        } else  if (signal) {
```
Comment (liamdebeasi) [inline packages/@ionic/utils-subprocess/src/index.ts]:
```suggestion
        } else if (signal) {
```

File: packages/@ionic/discover/src/publisher.ts [hunk 2]
```diff
@@ -173,6 +173,8 @@ export function prepareInterfaces(interfaces: { [index: string]: os.NetworkInter
 
       return false;
     });
+  }
+
```
Comment (liamdebeasi) [inline packages/@ionic/discover/src/publisher.ts]:
```suggestion
```

**Review by liamdebeasi — APPROVED** [review]
Two whitespace fixes, but otherwise this looks good.

**Comment by jcesarmobile** [comment]
I would have expected `npm run lint:fix` to fix those, but `npm run lint` just returns true, so it does nothing.
good catch!

**Pull request merged**